### PR TITLE
libechonest 2.3.1

### PR DESCRIPTION
**Pull request by Noctem**
Update to latest stable release, modernize formula.

### CODE REVIEW

File: Library/Formula/libechonest.rb [hunk 1]
```diff
@@ -13,14 +11,14 @@ class Libechonest < Formula
...
+  depends_on "qjson"
 
-  conflicts_with 'doxygen', :because => "cmake fails to configure build."
+  conflicts_with "doxygen", :because => "cmake fails to configure build."
 
   def install
     system "cmake", ".", *std_cmake_args
-    system "make install"
+    system "make", "install"
   end
```
Comment (mikemcquaid) [inline Library/Formula/libechonest.rb]: If you're feeling lovely: any chance a test could be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Comment (Noctem) [inline Library/Formula/libechonest.rb]: @mikemcquaid This only installs a dylib and headers.

Comment (mikemcquaid) [inline Library/Formula/libechonest.rb]: Yup, would be good if (like `tinyxml2.rb`) we could link import a header and link against the library.

Comment (Noctem) [inline Library/Formula/libechonest.rb]: I guess I skimmed over the last line of your original comment, my apologies.

File: Library/Formula/libechonest.rb [hunk 2]
```diff
@@ -13,14 +11,31 @@ class Libechonest < Formula
...
-  conflicts_with 'doxygen', :because => "cmake fails to configure build."
+  conflicts_with "doxygen", :because => "cmake fails to configure build."
 
   def install
     system "cmake", ".", *std_cmake_args
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
```
Comment (Noctem) [inline Library/Formula/libechonest.rb]: @mikemcquaid Does this look okay?

Comment (mikemcquaid) [inline Library/Formula/libechonest.rb]: Looks great!

**Comment by tdsmith** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Pull request closed without merging**